extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest: a global variable set true in Optimizely for 50% of users who visit ja wikia.
+ * window.wgGoogleSearchParam: Search ID set in Optimizely.
+ */
+$(function () {
+	if ( window.wgGoogleSearchTest ) {
```

Comment:
Please consider avoiding usage of globals. You could wrap the code in `require` closure and add https://github.com/Wikia/app/blob/dev/resources/wikia/modules/window.js as dependency. For example:

```JS
require(['wikia.window', 'wikia.location', 'jquery'], function (win, loc, $) {
 /// your code goes here
});
```